src/test/resources/config/bootstrap.yml

```diff
@@ -1,8 +1,9 @@
 spring:
```

Comment:
Only 1 line diff, but whole file is shown (indentetion)

Reply:
fiexed